databases/week2/diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE DATABASE diagram_db
+USE diagram_db
+-- -----------------------------------------------------
+-- Table`Products`
+-- -----------------------------------------------------
+CREATE TABLE Products (
+  `product_id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `item is stock` VARCHAR(45) NOT NULL,
```

Comment:
this does not belong to the class product
You could have an inventory table where you could have the product id, and the number of items in the stock